### PR TITLE
Add Fluid Slider

### PR DESCRIPTION
**Pull request by serhii-londar**
## Project URL
https://github.com/Ramotion/fluid-slider

## Description
A slider widget with a popup bubble displaying the precise value selected.

## Checklist

- [x] Only one project/change is in this pull request
- [ ] Addition in chronological order (bottom of category)
- [x] Supports iOS 9 / tvOS 10 or later
- [x] Supports Swift 4 or later
- [x] Has a commit from less than 2 years ago
- [x] Has a **clear** README in English

### CODE REVIEW

**Comment by danger-awesome-ios** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Found 2 link issues, a project collaborator will take care of these, thanks :)</td>
    </tr>
  </tbody>
</table>

#### Link issues by [`awesome_bot`](https://github.com/dkhamsing/awesome_bot)

  Line | Status | Link
| ---: | :----: | --- |
1269 | [301](https://httpstatuses.com/301) | https://creativesdk.adobe.com/ redirects to<br>https://www.adobe.io/apis/creativecloud/creativesdk.html
2763 | Error | https://www.mikeash.com/pyblog/ <br> Cannot assign requested address - connect(2) for "www.mikeash.com" port 443


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by lfarah** [comment]
Ramotion libs ❤️ 
Thanks for contributing 👍

File: README.md [hunk 1]
```diff
@@ -2166,6 +2166,7 @@ Most of these are paid services, some have free tiers.
 * [MultiSlider](https://github.com/yonat/MultiSlider) - UISlider clone with multiple thumbs and values, optional snap intervals, optional value labels. :large_orange_diamond:
 * [AGCircularPicker](https://github.com/agilie/AGCircularPicker) - AGCircularPicker is helpful component for creating a controller aimed to manage any calculated parameter. :large_orange_diamond:
 * [VSVerticalSlider](https://github.com/vsmithers1087/VSVerticalSlider) - An animatable and customizable vertical slider written in Swift4.🔶
+* [Fluid Slider](https://github.com/Ramotion/fluid-slider) - A slider widget with a popup bubble displaying the precise value selected.
```
Comment (lfarah) [inline README.md]: Please add ```:large_orange_diamond:``` in the end because it is a Swift lib

**Comment by serhii-londar** [comment]
@lfarah just added.

**Pull request merged**